### ISSUE TITLE
code and tests should run under python 3

### ISSUE THREAD

**Issue by nden** [STScI-JWST/jwst#1033]
Starting with astropy v 3.0 (some time at the end of 2017) astropy will not support python 2.x any more. While the jwst code has already been converted to run under python 3 and Travis is testing with python 3 it will be nice to run the nightly tests under v3 as well. For this to happen either pandokia needs to run under v 3 or we need to switch the nightly tests to a different system.

List of items that need to happen:
[] remove `six` from code - currently it's imported from `astropy`

**Comment by pllim** [comment]
Here is what happened when JWST tests were run with pandokia under Python 3: https://ssb.stsci.edu/pandokia/pandokia.cgi?query=summary&qid=3145

Note that only 8 tests (out of 100) made it. The other 92 were plagued by incompatibility. @jhunkeler had just fixed the "print statement" problem today, so I'll update more tomorrow.

**Comment by pllim** [comment]
Much better after "print statements" were fixed, still some failures but only several:

https://ssb.stsci.edu/pandokia/pandokia.cgi?query=summary&qid=3185

https://ssb.stsci.edu/pandokia/pandokia.cgi?query=summary&qid=3186

**Comment by hbushouse** [comment]
Yes, much better. The tests run under python 3 now show identical results to those under python 2 (the same tests showed the same errors/failures). Early this morning I updated the comparison files for the tests that showed "failure" (differences in results), so we should get an even closer to perfect results once today's report is available. At the very least, we've confirmed compatibility between the python 2 and 3 versions.
